### DISCUSSION TITLE
Forwarded logs

### DISCUSSION THREAD

**Discussion by Kofl** [anonaddy/anonaddy#407] [Feature Requests]
Hi,

maybe I missed it, but it would be super useful to have the information of the e.g. last 5 forwarded mails for a specific alias:

e.g. Date/Time, Sender, Subject

T

**Comment by willbrowningme** [comment]
Thanks for the feedback, if this was added it would obviously have to be opt-in.

**Reply by Kofl** [reply]
Yes, absolutely opt-in.

**Comment by Arpi** [comment]
Dear Will, 

Indeed, this would be useful. 
Indicate the date and time of the last forwarded or/and last blocked message for every alias. (The additional column may be displayed/hidden based on a setting on the Settings page.) 
And just like with other columns, make it possible to set it as the sorting order. 

If you think that the database could not handle the additional data (I suppose it would not be that much additional data), then at least save the last 5 deliveries/blocks. 
In this case, next to the tab “Failed Deliveries” there could be another tab called “Last emails” or “Last deliveries/blocks” with the last few (at least 5) emails forwarded or blocked. 
Obviously, keeping the date of the last forwarded/blocked email for every alias would be more useful. 

Thanks for this brilliant service, 
Arpad
